src/widgets/refinement-list/__tests__/refinement-list-test.js

```diff
@@ -27,6 +27,32 @@ describe('refinementList()', () => {
         refinementList(options);
       }).toThrow(/^Usage:/);
     });
+
+    it('throws an exception when showMoreLimit without showMore option', () => {
+      expect(
+        refinementList.bind(null, {
+          attribute: 'attribute',
+          container,
+          showMoreLimit: 10,
+        })
+      ).toThrowErrorMatchingInlineSnapshot(
+        `"\`showMoreLimit\` must be used with \`showMore\` set to \`true\`."`
```

Comment:
I curious why you prefer not let jest handle the snapshots here with `expect(...).toThrowErrorMatchingSnapshot()`.
(Since we're already using snapshots everywhere)

Reply:
This does let Jest handle the snapshot for us, except that it inlines it in the test file (and doesn't output it to a `.snap` file).

I find it handy for one-line snapshots – it avoids context switches when you need to open another file to read the output.